src/imitation/policies/serialize.py

```diff
@@ -72,14 +79,43 @@ def _load_stable_baselines(
         A function loading policies trained via cls.
     """
 
-    def f(path: str, venv: vec_env.VecEnv) -> policies.BasePolicy:
+    def f(venv: vec_env.VecEnv, path: str) -> policies.BasePolicy:
         """Loads a policy saved to path, for environment env."""
         model = load_stable_baselines_model(cls, path, venv)
         return getattr(model, "policy")
 
     return f
 
 
+def _load_stable_baselines_from_huggingface(
+    algo_name: str,
+    cls: Type[base_class.BaseAlgorithm],
+) -> PolicyLoaderFn:
+    """Higher-order function, returning a policy loading function.
```

Comment:
This docstring is almost identical to the one for `_load_stable_baselines_from_file`: add one or two sentences to the docstrings?

Reply:
Done